### PR TITLE
Fix: link updates

### PR DESCRIPTION
**Pull request by luisgruson**
Update links on docs

### CODE REVIEW

File: docs/kuneiform/sql-as-understood-by-kwil/scalar-functions.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ The glob(X,Y) function is equivalent to the expression "Y GLOB X". Note that the
 name GLOB '*helium*'
 glob('*helium*',name) 
 ```
+    name GLOB '*helium*'
```
Comment (KwilLuke) [inline docs/kuneiform/sql-as-understood-by-kwil/scalar-functions.md:34]: Please double check this, not sure if the code block should appear twice.

**Review by KwilLuke — CHANGES_REQUESTED** [review]

**Comment by KwilLuke** [comment]
@luisgruson Can you please double check that **all** the relative links are fixed? I am still seeing quite a few in this PR:
relative-paths

**Comment by KwilLuke** [comment]
@luisgruson Can you please double check that **all** the relative links are fixed? I am still seeing quite a few in this PR:
relative-paths

**Comment by luisgruson** [comment]
Made the updates to the absolute links. I kept relative links if they tie to assets on the same page (syntax tree)

**Review by KwilLuke — APPROVED** [review]

**Pull request merged**